src/main/java/net/minestom/server/network/packet/client/login/LoginStartPacket.java

```diff
@@ -59,7 +57,7 @@ public void process(@NotNull PlayerConnection connection) {
             byte[] nonce = new byte[4];
             ThreadLocalRandom.current().nextBytes(nonce);
             socketConnection.setNonce(nonce);
-            socketConnection.sendPacket(new EncryptionRequestPacket("", publicKey, nonce));
+            socketConnection.sendPacket(new EncryptionRequestPacket(MinecraftServer.configuration().serverId(), publicKey, nonce));
```

Comment:
Pretty sure that server ID is meant to be empty.

Reply:
Well it's sent to the session server as an SHA-1 hash in the form of serverID + serverPubKey + clientPrivKey

Reply:
Yeah I know, the only reason I pointed this out is because vanilla uses a blank server ID, and it's standard to use a blank server ID

Reply:
I believe it used to be configurable in vanilla at some point, anyways i'm not sure what to do with this, since it has no real effect and having one more field in the config doesn't hurt either.